src/mame/drivers/djmain.cpp

```diff
@@ -1531,6 +1531,27 @@ ROM_START( bm3rdmix )
 ROM_END
 
 ROM_START( bmcompmx )
+	ROM_REGION( 0x100000, "maincpu", 0 )        /* MC68EC020FG25 MPU */
+	ROM_LOAD16_BYTE( "858jac01.6a", 0x000000, 0x80000, CRC(b32693ca) SHA1(6518a8acbd070bb6f9039c4f9997dda2720e8e16) )
+	ROM_LOAD16_BYTE( "858jac02.8a", 0x000001, 0x80000, CRC(0d11d0f4) SHA1(b9fbf2ce062e1ee4a785abe021ac039857ced0d2) )
```

Comment:
Only these two ROMs changed. The other ROM files are the same as the previous JAB build.